app/templates/components/ui-table/cell/events/view/videoroom/cell-stream-title.hbs

```diff
@@ -1,6 +1,6 @@
-<div class="d-flex">
+<div class="ml-auto">
   {{@record}}
-  <div class="ml-auto">
+  <div class="d-flex">
```

Comment:
This just looks like a swap for no good reason. This is not hit and trial. Every change should have a reason

Reply:
But I tried out with various way. Sometime I failed with mobile view so this logic became valid for me. Thanks. Any suggestions?